### ISSUE TITLE
New maDMP examples

### ISSUE THREAD

**Issue by TomMiksa** [RDA-DMP-Common/hackathon-2020#2]
Here is a collection of DMPs and maDMPs:
https://zenodo.org/communities/tuw-dmps-ds-2020/

Let's review them and create some examples that all of us can use for testing our implementation of the standard.

**Comment by sjDCC** [comment]
Sounds good. Perhaps a common activity could be designing an import function so this collection of maDMPs could be added to and rendered in the various tools e.g. DMPonline, DMPTool, Data Stewardship Wizard, EasyDMP etc. 

Thoughts @MarekSuchanek @hmpf @briri @xsrust @TomMiksa 

Links to tickets #1 and #8

**Comment by briri** [comment]
Yes, we've begun work on an [api endpoint](https://github.com/DMPRoadmap/roadmap/blob/development/app/controllers/api/v1/plans_controller.rb) that will accept the RDA common metadata standard (in json format) so these would be an excellent test. We also have one that will export a DMP in that format as well.

**Comment by gkakas** [comment]
We are also working on our API for importing/exporting maDMPs and we will enjoy working together validating, refining an extending our implementation so as to expand the coverage of the specification
#21
